src/rgw/rgw_process.cc

```diff
@@ -354,6 +355,7 @@ int process_request(rgw::sal::RGWRadosStore* const store,
   if (handler)
     handler->put_op(op);
   rest->put_handler(handler);
+  *latency = s->time_elapsed();
```

Comment:
* if `latency` is a pointer, we should handle the nullptr case
* the `dout()` just below calls `s->time_elapsed()` again; let's avoid calling it twice, because reading the clock isn't free

```c++
  const auto lat = s->time_elapsed();
  if (latency) {
    *latency = lat;
  }
  dout(1) << "====== req done req=" << hex << req << dec
	  << " op status=" << op_ret
	  << " http_status=" << s->err.http_ret
	  << " latency=" << lat
```